lib/onigumo.ex

```diff
@@ -3,23 +3,29 @@ defmodule Onigumo do
   Web scraper
   """
   @input_filename "urls.txt"
+  def input_filename, do: @input_filename
   @output_filename "body.html"
+  def output_filename, do: @output_filename
 
   def main() do
     HTTPoison.start()
+    save_URLs_contents(@input_filename, @output_filename)
+  end
+
+  def save_URLs_contents(input_file, output_file) do
```

Comment:
Let’s stick to Elixir’s own snake case style and tread acronyms as normal words, keeping them lowercase. Examples: [`Kernel.is_pid`](https://hexdocs.pm/elixir/1.7.0-rc.1/Kernel.html#is_pid/1) for PID, [`Calendar.naive_datetime_from_iso_days`](https://hexdocs.pm/elixir/1.7.0-rc.1/Calendar.html#c:naive_datetime_from_iso_days/1)

```suggestion
  def save_urls_contents(input_file, output_file) do
```

Reply:
sure good point